int32 Tag_id

int32 Anchor_available 
#00001111 = 15: 4 Anchors(A0~A3)
#00000111 = 7 : 3 Anchors(A0~A2)
#00000011 = 3 : 2 Anchors(A0~A1)

float32 to_A0
float32 to_A1
float32 to_A2
float32 to_A3
